### PR TITLE
18 ai create methods collecting statictical data of input dataset

### PR DESCRIPTION
**Pull request by JakubCzyzewski-TomTom**

### CODE REVIEW

File: backend/ai/views.py [hunk 1]
```diff
@@ -19,16 +21,24 @@ def prediction():
...
 def get_average_comments():
     return stats.load_data('VideoCommentCount', "Average Comments")
 
+
 @ai_blueprint.route("/statistics/views")
 def get_average_views():
     return stats.load_data('videoViewCount', "Average Views")
 
 
+@ai_blueprint.route("statistics/describe")
```
Comment (matkrasinski) [inline backend/ai/views.py:40]: missing "/" before statistics/describe

**Review by MichalAndrzejczak1 — APPROVED** [review]

File: backend/ai/views.py [hunk 2]
```diff
@@ -19,16 +21,24 @@ def prediction():
 def get_average_likes():
     return stats.load_data('videoLikeCount', "Average Likes")
 
+
 @ai_blueprint.route("/statistics/dislikes")
 def get_average_dislikes():
     return stats.load_data('videoDislikeCount', "Average Dislikes")
 
+
 @ai_blueprint.route("/statistics/comments")
```
Comment (matkrasinski) [inline backend/ai/views.py:30]: add methods arg with http method

File: backend/ai/views.py [hunk 3]
```diff
@@ -19,16 +21,24 @@ def prediction():
...
 def get_average_comments():
     return stats.load_data('VideoCommentCount', "Average Comments")
 
+
 @ai_blueprint.route("/statistics/views")
```
Comment (matkrasinski) [inline backend/ai/views.py:35]: same as above

File: backend/ai/views.py (hunk 1 above)
Comment (matkrasinski) [inline backend/ai/views.py:40]: add http methods

File: backend/ai/views.py [hunk 4]
```diff
@@ -19,16 +21,24 @@ def prediction():
 def get_average_likes():
     return stats.load_data('videoLikeCount', "Average Likes")
 
+
 @ai_blueprint.route("/statistics/dislikes")
```
Comment (matkrasinski) [inline backend/ai/views.py:25]: same as above

File: backend/ai/views.py [hunk 5]
```diff
@@ -2,8 +2,10 @@
 
 from .data import stats
 from ..ai.use import test
+from ..stats.StatCalculator import StatCalculator
 
 ai_blueprint = Blueprint("ai", __name__, url_prefix="/ai")
+stat_calc = StatCalculator()
 
 
 @ai_blueprint.route("/predict", methods=['POST'])
```
Comment (matkrasinski) [inline backend/ai/views.py:11]: same as above

**Pull request merged**